### PR TITLE
Added ability to get thumburl from MediaSelector

### PR DESCRIPTION
**Pull request by ndunand**

### CODE REVIEW

File: statics/media_selector.js [hunk 1]
```diff
@@ -57,13 +57,17 @@ window.MediaSelector.prototype.init = function() {
...
-        obj.onPick(data.item.oid, data.initial_pick);
+        obj.onPick(data.item.oid, data.item.thumb, data.initial_pick);
     }, false);
 };
 
-window.MediaSelector.prototype.onPick = function(oid) {
+window.MediaSelector.prototype.onPick = function(oid, thumburl) {
     var input = document.querySelector('#' + this.target + ' #id_mediaid');
     input.value = oid;
+    var input_thumb = document.querySelector('#' + this.target + ' input[name=mediaimg]');
```
Comment (sdiemer) [inline statics/media_selector.js:67]: On our Moodle, there is no input matching `input[name=mediaimg]`, is this related to a custom Moodle theme ?

Comment (ndunand) [reply statics/media_selector.js:67]: Hi @sdiemer , this is actually a new thing in https://github.com/UbiCastTeam/moodle-atto_ubicast/blob/main/mod_form.php#L46 .

Comment (sdiemer) [reply statics/media_selector.js:67]: Thank you for your response, I will merge this request.
